### PR TITLE
fix(list.tsx):excel export with good names in columns

### PR DESCRIPTION
**Pull request by Fenerz07**

### CODE REVIEW

**Review by Maxime-Labbe — CHANGES_REQUESTED** [review]
Too much upper case need to change this asap

File: components/list.tsx [hunk 1]
```diff
@@ -105,19 +105,189 @@ const List: React.FC<{
...
+                    heure_reception: 'Heure de réception',
+                    nombre_palettes_recues: 'Nombre de palettes reçues',
+                    nombre_palettes_consignees_recues:
+                        'Nombre de palettes consignées reçues',
+                    nombre_palettes_consignees_rendues:
+                        'Nombre de palettes consignées rendues',
+                    nombre_cartons_recus: 'Nombre de cartons reçus',
+                    poids_recu_kg: 'Poids reçu en kg',
+                    code_Prestataire: 'Prestataire',
+                    code_type_de_Prestataire: 'Type de Prestataire',
```
Comment (MathisDacacio) [inline components/list.tsx:201]: here

File: components/list.tsx [hunk 2]
```diff
@@ -105,19 +105,189 @@ const List: React.FC<{
...
+                    adresse: 'Adresse',
+                    civilite_contact_prestataire: 'Civilité du prestataire',
+                    nom_contact_prestataire: 'Nom du prestataire',
+                    prenom_contact_prestataire: 'Prénom du prestataire',
+                    telephone_contact_prestataire: 'Téléphone du prestataire',
+                    mail_contact_prestataire: 'Mail du prestataire',
+                    date_arret_activite_du_prestataire:
+                        "Date d'arrêt d'activité du prestataire",
+                    code_Utilisateur_Prospecteur: 'Utilisateur Prospecteur',
+                    code_Entite_Prospectee: 'Entité Prospectée',
```
Comment (MathisDacacio) [inline components/list.tsx:217]: here

File: components/list.tsx [hunk 3]
```diff
@@ -105,19 +105,189 @@ const List: React.FC<{
...
+                    nom: 'Nom',
+                    prenom: 'Prénom',
+                    tel_perso: 'Téléphone personnel',
+                    password: 'Mot de passe',
+                    code_Societe: 'Société',
+                    Logo: 'Logo',
+                    site_Web: 'Site Web',
+                    code_type_activite_Societe: "Type d'activité de la société",
+                    code_Groupe_appartenance: 'Appartenance du Groupe',
+                    date_arret_activite_Societe:
```
Comment (MathisDacacio) [inline components/list.tsx:245]: here

File: components/list.tsx [hunk 4]
```diff
@@ -105,19 +105,189 @@ const List: React.FC<{
...
+                        "Date d'arrêt d'activité de la société",
+                    code_type_de_Site: 'Type de site',
+                    code_site_suivi: 'Suivi de site',
+                    code_utilisateur_suivant: 'Utilisateur suivant',
+                    code_groupe: 'Groupe',
+                    nom_du_Groupe: 'Nom du Groupe',
+                    date_arret_activite_du_Groupe:
+                        "Date d'arrêt d'activité du Groupe",
+                    code_entite: 'Entité',
```
Comment (MathisDacacio) [inline components/list.tsx:254]: here

**Review by MathisDacacio — COMMENTED** [review]
fix

File: components/list.tsx [hunk 5]
```diff
@@ -105,19 +105,189 @@ const List: React.FC<{
...
+                    code_type_utilisateur: "Type d'utilisateur",
+                    date_fin_activite: "Date de fin d'activité",
+                    civilite: 'Civilité',
+                    nom: 'Nom',
+                    prenom: 'Prénom',
+                    tel_perso: 'Téléphone personnel',
+                    password: 'Mot de passe',
+                    code_Societe: 'Société',
+                    Logo: 'Logo',
+                    site_Web: 'Site Web',
```
Comment (Maxime-Labbe) [inline components/list.tsx:242]: here

**Review by Maxime-Labbe — APPROVED** [review]
LGTM except the comment

**Pull request merged**